### PR TITLE
[TESTES] Instalar jest e configurar TravisCI para testes de front-end

### PR DESCRIPTION
**Pull request by markx3**
Hi,

Here we install `Jest` and create the first test for our `Dashboard` component. Also, `.travis.yml` was refactored so we can have automated tests run on TravisCI.

### CODE REVIEW

**Comment by markx3** [comment]
[[TESTES] Instalar jest e configurar TravisCI para testes de front-end (1) [1]](https://trello.com/c/01R1knDN/26-testes-instalar-jest-e-configurar-travisci-para-testes-de-front-end-1-1)

File: .travis.yml [hunk 1]
```diff
@@ -1,22 +1,40 @@
...
+cache:
+  yarn: true
+  directories:
+    - '.eslintcache'
+    - 'node_modules'
+
+install:
+  - bundle install
+  - nvm install node
+  - node -v
```
Comment (cuzik) [inline .travis.yml]: We need show the version?

File: Gemfile [hunk 2]
```diff
@@ -18,6 +18,9 @@ gem 'turbolinks', '~> 5'
 gem 'jbuilder', '~> 2.5'
 gem 'mini_racer'
 gem 'sass-rails', '~> 5.0'
+gem 'bootstrap', '~> 4.1.1'
+gem 'material-sass', '~> 4.1.1'
+gem 'jquery-rails'
```
Comment (cuzik) [inline Gemfile]: alphabetical order <3

File: package.json [hunk 3]
```diff
@@ -3,13 +3,39 @@
   "private": true,
   "dependencies": {
     "@rails/webpacker": "3.5",
+    "@vue/test-utils": "^1.1.alpha.4",
+    "babel-jest": "^23.4.2",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-es2017": "^6.24.1",
```
Comment (cuzik) [inline package.json]: We need `es2017` too?

File: app/javascript/packs/dashboard/__tests__/index.spec.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+  beforeEach(() => {
+    localVue = createLocalVue()
+    localVue.use(Vuetify)
+  })
+
+  it('renders the Dashboard component', () => {
+
+    const wrapper = shallowMount(Dashboard, { localVue })
+    expect(wrapper).toMatchSnapshot()
```
Comment (cuzik) [inline app/javascript/packs/dashboard/__tests__/index.spec.js]: separate expect from `const`

File: .travis.yml (hunk 1 above)
Comment (markx3) [reply .travis.yml]: I used the `webpacker` repo as reference for this file. I'm not sure why they do this. :)

Comment (cuzik) [reply .travis.yml]: :smile_cat:

**Pull request merged**